projects/packages/forms/src/contact-form/css/grunion-admin.css

```diff
@@ -148,26 +148,6 @@
 }
 
 /* override JP logo */
```

Comment:
We can remove this comment and the above two: `feedback-export-modal__footer` and `feedback-export-modal__footer-column`.